examples/advanced.mjs

```diff
@@ -0,0 +1,36 @@
+import * as mg from '../modules/mgclient.mjs';
+
+function cleanUp(mgClient) {
+    mgClient.destroySession();
+    mg.finalize();
+}
```

Comment:
`mgClient` could have a "close" method where the `destroySession` method will be called, also `mg` could have a "close". Is it possible to do that somehow without any user actions?

Reply:
I rather not, these two are decoupled because `finalize()` also frees the resources managed by wasm. In the long run we should find the most generic way to do this.